content/en/docs/guides/metadata.md

```diff
@@ -0,0 +1,101 @@
+---
+title: Metadata
+description: >-
+  Explains what metadata is, how it is transmitted, and what it is used for.
+---
+
+### Overview
+
+Metadata is a side channel that allows clients and servers to provide
+information to each other that is associated with the data messages.
+
+gRPC metadata is a key-value pair of data that is sent with a gRPC request or
+response. It is used to provide additional information about the call, such as
+authentication credentials, tracing information, or custom headers.
+
+gRPC metadata is implemented using HTTP/2 headers. The keys and values are ASCII
+strings, but the values can also be binary data. The keys are case insensitive
+and must not start with the prefix grpc-, which is reserved for gRPC itself.
+
+gRPC metadata can be sent and received by both the client and the server. It is
+sent in the initial headers of the RPC call, and can also be sent in subsequent
```

Comment:
I don't think this is clear when these things happen. I don't know if you are referring to client-side or server-side for the initial headers, for example.

Reply:
reworded